src/components/change-requests/change-request-details/change-request-details/type-specific-details/stage-gate-details/stage-gate-details.test.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from "@testing-library/react";
+import { StageGateChangeRequest } from "utils";
+import { exampleStageGateChangeRequest } from "../../../../../../test-support/test-data/change-requests.stub";
+import StageGateDetails from "./stage-gate-details";
+
+/**
+ * Sets up the component under test with the desired values and renders it.
+ */
+const renderComponent = (cr: StageGateChangeRequest) => {
+  return render(<StageGateDetails cr={cr} />);
+}
+
+describe('Change request details stage gate cr display element tests', () => {
+  it('Renders confirm completed', () => {
+    const cr: StageGateChangeRequest = exampleStageGateChangeRequest;
+    renderComponent(cr);
+    expect(screen.getByText(`Confirm WP Completed`)).toBeInTheDocument();
+    expect(screen.getByText(`${cr.confirmDone ? 'YES' : 'NO'}`)).toBeInTheDocument();
+  });
+
+  it('Renders leftover budget', () => {
+    const cr: StageGateChangeRequest = exampleStageGateChangeRequest;
```

Comment:
You could penalty just lift this line to be the first line inside the describe so as to eliminate having this line twice

Reply:
I'll look into it and check the other tests to see if there is anything else that can be refactored.